templates/configurations/form.interface.hbs

```diff
@@ -84,6 +84,37 @@
       </div>
     {{/hasOptions}}
 
+     <div v-if="_componentsWithDescription.includes(_componentName)" class="form-group cleafix">
+       <div class="col-xs-12">
+        <label>Description</label>
+        <input class="form-control" type="text" v-model.trim="description" placeholder="Description text" />
+      </div>
+    </div>
+
+    <div v-if="_componentsWithPersonalization.includes(_componentName)" class="form-group cleafix">
+      <div class="col-xs-12">
+        <label>Default value type</label>
+        <select class="form-control" :value="source" v-model="source">
+          <option value="default">Enter a value</option>
+          <option value="profile">User profile data</option>
+          <option value="appStorage">App storage variable</option>
+          <option value="query">Link query parameter</option>
+        </select>
+        <div class="col-xs-12 text-right">
+          <a href="#!" target="_blank">Learn more about these types</a>
+        </div>
+        <div v-show="source !== 'default'">
+          <label>Default key</label>
+          <div v-show="!key" class="text-danger">
+            This field is required*
+          </div>
+          <input v-bind:class="{ 'border-danger': !key }" v-show="source === 'profile'"class="form-control" type="text" v-model.trim="key" placeholder="Enter user data field/column name" />
```

Comment:
any specific reason for using `v-show` instead of `v-if`?

Reply:
@squallstar No, should we use `v-if` instead?

Reply:
https://v1.vuejs.org/guide/conditional.html

> The difference is that an element with v-show will always be rendered and remain in the DOM; v-show simply toggles the display CSS property of the element.

As you can see, `v-if` is better because the node only gets rendered as required. On the other hand, use `v-show` when removing and adding the node can result in problems (e.g. because of added plugins on top of the rendering).